lib/quicly.c

```diff
@@ -3325,7 +3327,7 @@ static int handle_ack_frame(quicly_conn_t *conn, size_t epoch, quicly_ack_frame_
     /* Update loss detection engine on ack. The function uses ack_delay only when the largest_newly_acked is also the largest acked
      * so far. So, it does not matter if the ack_delay being passed in does not apply to the largest_newly_acked. */
     quicly_loss_on_ack_received(&conn->egress.loss, largest_newly_acked.packet_number, now, largest_newly_acked.sent_at,
-                                frame->ack_delay, bytes_acked);
+                                frame->ack_delay, largest_newly_acked.ack_eliciting);
```

Comment:
A minor fix: I don't think we want to know if just the largest_newly_acked was ack-eliciting. We want to know if _any_ newly acked was ack-eliciting.

Reply:
Absolutely. Fixed in a30516e. Please review.